lib/src/Base/Optim/OptimizationSolverImplementation.cxx

```diff
@@ -30,6 +30,7 @@ OptimizationSolverImplementation::OptimizationSolverImplementation()
   : PersistentObject()
   , startingPoint_(NumericalPoint(0))
   , maximumIterationsNumber_(ResourceMap::GetAsUnsignedInteger( "OptimizationSolverImplementation-DefaultMaximumIteration" ))
+  , maximumEvaluationNumber_(ResourceMap::GetAsUnsignedInteger("OptimizationSolverImplementation-DefaultMaximumEvaluationNumber"))
```

Comment:
We should be consistent: either maximumIterationsNumber_/maximumEvaluationsNumber_ or maximumIterationNumber_/maximumEvaluationNumber_, I am not sure which one is right, it seems that we prefer singular form.